changelog/unreleased/3381

```diff
@@ -0,0 +1,7 @@
+Bugfix: Account removed is not removed from the drawer
+
+When an account was deleted from the phone settings, in the accounts section, it was not deleted from the Navigation Drawer,
+now when deleting an account from there, the Navigation Drawer refreshes and it is deleted correctly.
```

Comment:
Just a tiny change I suggest, but up to you 😉 
P.S.: I like the passive voice you used 👍 
```suggestion
When an account was deleted from the app settings, in the accounts section, it was not removed from the Navigation Drawer.
Now, when deleting an account from there, the Navigation Drawer is refreshed and the removed account is no more shown.
```

Reply:
Oops, you're wrong my friend 😄 , the problem appears when the user deletes his account from the phone settings (not from the app settings)

Thanks for the other suggestions, no problem to change them.

Reply:
😱  My fault!! Sorry, thanks for the information